cmd/create-tap-device.go

```diff
@@ -49,7 +66,19 @@ func createTapDeviceOnPIDNetNs(launcherPid string, tapName string, uid uint, gid
 		glog.V(4).Info("Successfully loaded netns ...")
 
 		err = netns.Do(func(_ ns.NetNS) error {
-			if err := createTapDevice(tapName, uid, gid, false); err != nil {
+			desiredLabel := "system_u:system_r:container_t:s0"
+			if err := selinux.SetExecLabel(desiredLabel); err != nil {
```

Comment:
please correct the commit  message, that you chose to use the alternative (=using `SetExecLabel` inside the `netns.Do()`)